IdentityCore/src/MSIDOAuth2Constants.h

```diff
@@ -117,6 +117,7 @@ extern NSString *const MSID_REFRESH_TOKEN_CACHE_TYPE;
 extern NSString *const MSID_LEGACY_TOKEN_CACHE_TYPE;
 extern NSString *const MSID_ID_TOKEN_CACHE_TYPE;
 extern NSString *const MSID_GENERAL_TOKEN_CACHE_TYPE;
+extern NSString *const MSID_GENERAL_CACHE_ITEM_TYPE;
```

Comment:
What is a general cache item?

Reply:
It is just a category that we can use to store non credential items. I added metadata type under general cache item type and in future we can add other generic keychain items to it.